toolbox/linux/nowwhat.rst

```diff
@@ -0,0 +1,127 @@
+I Installed It!  Now What?
+==========================
+
+Congratulations! If everything worked properly, you should be looking at a
+freshly installed Linux login screen, either inside a virtual machine or "for
+real for real" on your main computer.
+
+What to do from here is up to you. The Toolbox pages after this one give some
+great things to learn to do on Linux. However, here are some other things you
+may want to do besides the technical stuff.
+
+*Note that the commands in code blocks below should be run in your Terminal
+program on Linux.  Also, these assume you installed Ubuntu.*
```

Comment:
Should we follow the convention of prepending shell commands with a $? It would make beginners more familiar when they encounter it on other blogs, StackOverflow, etc. We should just make a note about what the $ signifies.